src/service/routes/users.js

```diff
@@ -29,4 +36,68 @@ router.get('/:id', async (req, res) => {
   res.send(user);
 });
 
+// Add SSH public key
+router.post('/:username/ssh-keys', async (req, res) => {
+  if (!req.user) {
+    res.status(401).json({ error: 'Authentication required' });
```

Comment:
I'm wondering if this could be `'Login required'` to make it a bit more clear that the user should login first. Since it will mainly be used as an API, the flow may not be obvious.

Feel free to ignore this if the flow (including login) is documented!